src/main/resources/swagger/api-docs.yaml

```diff
@@ -599,6 +599,41 @@ paths:
           description: Submission or Workflow not found
         500:
           description: Internal Error
+          
+  /workspaces/{workspaceNamespace}/{workspaceName}/updateAttributes:
+    get:
+      tags:
+      - Workspaces
```

Comment:
This introduces a new header section in the swagger UI. Could you also update the other "workspaces" tags (lowercased) with the upper case version as part of this? It would be purely for the sake of consistency.